book/src/integrations/ldap.md

```diff
@@ -69,6 +69,11 @@ As the POSIX password is not equivalent in strength to the primary credentials o
 most cases is multi-factor authentication), the LDAP bind does not grant rights to elevated read
 permissions. All binds have the permissions of "anonymous" even if the anonymous account is locked.
 
+> As the POSIX password is not equivalent in strength to the primary credentials of Kanidm
+
+This is no longer necessarily true (thanks to the fallback). Would it be possible to  enable elevated read permissions
```

Comment:
If the primary cred is pw+totp, then ldap is pw only so it's still not equal security wise.